numpy/linalg/umath_linalg.c.src

```diff
@@ -651,39 +651,39 @@ CDOUBLE_mulc(DOUBLECOMPLEX_t op1, DOUBLECOMPLEX_t op2)
     return result;
 }
 
-static inline void
+static NPY_INLINE void
 print_FLOAT(npy_float s)
 {
     TRACE_TXT(" %8.4f", s);
 }
-static inline void
+static NPY_INLINE void
 print_DOUBLE(npy_double d)
 {
     TRACE_TXT(" %10.6f", d);
 }
-static inline void
+static NPY_INLINE void
 print_CFLOAT(npy_cfloat c)
 {
     float* c_parts = (float*)&c;
```

Comment:
Yes, we should be able to automate things to some extent. There are some decent tools out there. However, once done it isn't too difficult to keep things from sliding.